.azure/linux-stack.yml

```diff
@@ -0,0 +1,43 @@
+jobs:
+- job: ${{ parameters.name }}
+  timeoutInMinutes: 0
+  pool:
+    vmImage: ${{ parameters.vmImage }}
+  strategy:
+    matrix:
+      stack-def:
+        BUILD: stack
+        STACK_YAML: stack.yaml
+  steps:
+  - script: |
+      git submodule sync
+      git submodule update --init
+    displayName: Sync submodules
+  - script: |
+      export STACK_ROOT="$(Build.SourcesDirectory)"/.stack-root
+      mkdir -p ~/.local/bin
+      curl -L https://get.haskellstack.org/stable/linux-x86_64.tar.gz | \
+        tar xz --wildcards --strip-components=1 -C ~/.local/bin '*/stack'
+    displayName: Install stack
+  - script: |
+      export PATH=$HOME/.local/bin:$PATH
+      stack --install-ghc build --only-dependencies
+    displayName: Build dependencies
+  - script: |
+      export PATH=$HOME/.local/bin:$PATH
+      stack build
+    displayName: Build `hie`
+  - script: |
+      export PATH=$HOME/.local/bin:$PATH
+      stack build --test --bench --only-dependencies
+      stack install # `hie` binary required for tests
+      # TODO: get `cabal` binary from somewhere else?
+      stack install cabal-install
+      stack exec hoogle generate
+      cabal v1-update
+      cabal v1-install liquidhaskell --symlink-bindir=$HOME/.local/bin
```

Comment:
@power-fungus It looks like this is installing liquidhaskell 0.6.0.0, but the one we use for testing is 0.8.2.4 I believe. Can you try `cabal new-install liquidhaskell-0.8.2.4`? Although I'm not sure what versions of ghc this builds with

Reply:
You need GHC 8.2.2 to install liquidhaskell. I just rebuild my GHC stuff, and installed with

    cabal new-install liquidhaskell -w ghc-8.2.2